### ISSUE TITLE
Connection Problem

### ISSUE THREAD

**Issue by lyons7777** [maxsokolov/GoogleKit#5]
I was using this lib, worked ok, but suddenly stopped bringing data... for any search term

**Comment by maxsokolov** [comment]
Hi @lyons7777 

Sounds like it's not problem with the lib. You also have to check your quota, Google's services may have limitations (requests per seconds etc).

**Comment by lyons7777** [comment]
@maxsokolov Hi i'm using 'lookupLocation' and is working perfect!
btw did you set as a deprecate lib?
Thanks!

**Comment by maxsokolov** [comment]
@lyons7777 yeah, sorry for that. It's pretty old project, so I'm not going to support it anymore. I have more exciting projects.

**Comment by lyons7777** [comment]
@maxsokolov ok this was awesome to, thanks!

**Comment by mkll** [comment]
@maxsokolov This "pretty old" project is still very useful, so please put the docs back to README.MD. ;)
